collect/usace/wcds.py

```diff
@@ -311,4 +312,196 @@ def get_reservoir_metadata(reservoir, water_year, interval='d'):
         result.update({'gross pool (elev)': metadata_dict['ymarkers']['Gross Pool(elev)']['value']})
     
     # returns relevant subset as dictionary
-    return result
\ No newline at end of file
+    return result
+
+
+def read_fcr(datetime_structure):
+    """
+    read the text in the Sacraamento Valley section of the USACE SPK FCR report
+
+    Arguments:
+        datetime_structure (datetime): datetime object
+    Returns:
+        (list of strings): list of strings from the FCR report
```

Comment:
Can you check spelling in the description here?  The argument type should be `(datetime.datetime)` and the return type I think would just be `(list)` here.

I also like the "extract" verb you used for the other methods so could you rename this to `extract_fcr_text`?